src/Datadog.Trace/Tracer.cs

```diff
@@ -586,7 +586,7 @@ internal async Task WriteDiagnosticLog()
                     writer.WriteEndObject();
                 }
 
-                Log.Information("DATADOG TRACER CONFIGURATION - {0}", stringWriter.ToString());
+                Log.Debug("DATADOG TRACER CONFIGURATION - {Configuration}", stringWriter.ToString());
```

Comment:
This is a special log message emitted by all language tracers. We want this to always be written, so it should not be debug level. It's also not a warning or error (those have negative connotations). This is probably one of the reasons why we had the `Information` level and that was the tracer's default level.

Reply:
Reverted this change back to Information